workspaces/jupyterlab-python/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/jupyter/minimal-notebook:2025-03-31
+FROM quay.io/jupyter/minimal-notebook:2025-04-10
```

Comment:
This is today's tag. I've seen cases in the past where the same tag was pushed more than once in a day, which means that (barring timezone issues) this could cause a different base image to be used in future builds. Should we wait until end of day to merge this? Nit, of course.

Reply:
I'll keep an eye on the tags as this merges + builds in case the base image changes.